### PR TITLE
Developed README file using GitHub MD syntax

### PR DESCRIPTION
**Pull request by IrynaBarna**
Signed-off-by: Iryna Barna <irynax.barna@intel.com>

---
This change is [Reviewable](https://reviewable.io/reviews/taf3/testcases/1)

### CODE REVIEW

**Comment by rbbratta** [comment]
@lijenni @esmacgil   testing at mentions, for reviews

**Comment by IrynaBarna** [comment]
Please review @rbbratta @seungweonpark @NazarTkachuk @orestkreminskyi

File: README.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
+## Directory Structure
```
Comment (orestkreminskyi) [inline README.md]: Suggest to add "Introduction" section. This section should contain brief description of repository (what this repository is about, the aim and purpose of this repository, etc...). Thus, user will have general overview of repo before moving to the details described in next sections.

Comment (IBarna) [inline README.md]: Done. Please, take a look.

**Comment by esmacgil** [comment]
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/taf3/testcases/1)*

**Comment by IrynaBarna** [comment]
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[README.md, line 1 at r1](https://reviewable.io:443/reviews/taf3/testcases/1#-Kc_ELzaFxo6bJkt221L-r1-1:-KeEPOe1RikK-6wCuwZ3:b-896fix) ([raw file](https://github.com/taf3/testcases/blob/d876edc91cf8a588ba0495d8923ec18186dbcd2e/README.md#L1)):*
<details><summary><i>Previously, IBarna wrote…</i></summary><blockquote>

Done. Please, take a look.
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/taf3/testcases/1)*

**Comment by seungweonpark** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/taf3/testcases/1#-:-KeEpuR-BYtofy13PqaT:bnfp4nl)*

**Pull request merged**